packages/react-native-session-replay/android/src/test/kotlin/com/datadog/reactnative/sessionreplay/utils/TextViewUtilsTest.kt

```diff
@@ -71,56 +64,30 @@ internal class ReactTextMapperTest {
         whenever(mockTextView.text).thenReturn(forge.aString())
         whenever(mockTextView.typeface).thenReturn(Typeface.SANS_SERIF)
 
-        whenever(
-            mockReactTextPropertiesResolver.addReactNativeProperties(
-                originalWireframe = eq(mockTextWireframe),
-                view = eq(mockTextView),
-                pixelDensity = eq(0f)
-            )
-        ).thenReturn(mockTextWireframe)
-
-        testedMapper = spy(
-            ReactTextMapper(
-                reactTextPropertiesResolver = mockReactTextPropertiesResolver
-            )
-        )
+        testedUtils = spy(TextViewUtils())
```

Comment:
Why this line is needed? I don't see any line where we would mock anything in the `testedUtils` instance.

Reply:
Yes, you're right. It was originally needed, but I must have missed it when cleaning up the file. I'll remove